.github/workflows/preview-storybook-deploy.yml

```
@@ -0,0 +1,139 @@
+    # only narrow, never widen), so this is also what each caller's job needs
+    # to declare. contents:read is for the checkout step below; id-token:write
+    # is required for the AWS OIDC step and is never part of the default
+    # GITHUB_TOKEN permissions.
+    permissions:
+      contents: read
+      id-token: write
+      pull-requests: write
+
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v3
```

actionlint (1.7.12)

[error] the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)